### PR TITLE
Adding new machines to 86Box

### PR DESCRIPTION
**Pull request by SauceJustSauce**
OK, so here it is.

New additions:

- IBM PS/ValuePoint 433DX/Si (SiS 471-ish)
- AST Adventure! 225 (430VX, uses a version of the BXB flash chip with the high pin (A16) inverted): Machine works as intended. 
- HP Pavilion 7320 (RU430HX-based, 2 Mbit): Just like every other 2 Mbit machine, it spits out the NVRAM error
- Sony Vaio PCV-90 (AG430HX-based): It (along with the TC430HX) seem to hang without an MPU-401 on port 0x300. It also spits out an NVRAM error (which is obvious).
- QDI PX483P3
- Packard Bell PB540 (Premiere/PCI LX-based)

Some machines would have incorrect PCI slots.

### CODE REVIEW

File: src/machine/machine_table.c [hunk 1]
```diff
@@ -26,28 +26,15 @@
 #include <string.h>
 #include <wchar.h>
 #include "../86box.h"
-#include "../cpu/cpu.h"
+#include "../cpu_new/cpu.h"
```
Comment (MoochMcGee) [inline src/machine/machine_table.c]: This change is incorrect. Please revert it.

File: src/machine/machine_table.c [hunk 2]
```diff
@@ -26,28 +26,15 @@
...
 #include "../mem.h"
 #include "../rom.h"
 #include "../device.h"
 #include "machine.h"
 
 
-#if defined(DEV_BRANCH) && defined(USE_AMD_K)
```
Comment (MoochMcGee) [inline src/machine/machine_table.c]: This change too.

**Comment by MoochMcGee** [comment]
Also, please rebase your changes onto latest master.

**Comment by SauceJustSauce** [comment]
> 
> 
> Also, please rebase your changes onto latest master.

Done.

**Comment by MoochMcGee** [comment]
That's... not rebasing but okay...

**Comment by MoochMcGee** [comment]
@SauceJustSauce Merging master into your pull request is not rebasing, and needlessly clutters the commit log.

**Pull request closed without merging**